module/actor/actor.js

```diff
@@ -147,28 +152,82 @@ export class yzecoriolisActor extends Actor {
     return chatOptions;
   }
 
-  _prepHPBonuses() {
-    // look through talents for any HPBonuses
+  _prepHPModifications() {
+    // look through items for any HP-Modifications
     let bonus = 0;
     for (let t of this.items) {
-      if (t.type !== "talent") {
-        continue;
-      }
-      const tData = t.system;
-      bonus += Number(tData.hpBonus);
+      const tData = t.system.itemModifiers;
+      bonus += Number(Object.keys(tData).reduce((counter,x) => {
+        counter += (tData[x].mod === "itemModifierHP")
```

Comment:
before I forget: need to add a check if gear/weapon/armor is equipped
_(for the other _preps too)_

Reply:
resolved with https://github.com/winks-vtt/yze-coriolis/pull/235/commits/b540b21218b4ad055f54e4b49cc38c3d45360030